### ISSUE TITLE
R&S Updates:  Redirect PDF forms article

### ISSUE THREAD

**Issue by lwalsh617** [department-of-veterans-affairs/va.gov-team#26704]
### Instructions
- Requests for URL changes/redirects need to be submitted **AT LEAST** 2 weeks in advance.  Some changes/redirects take significant amount of time and coordination to implement, so start the process as soon as you know you will need one. 
- This issue will be used from initial request through implementation of the redirect to ensure all individuals working on this are notified of status updates.  Please do not create multiple issues to track differnt steps.
- It is your responsibility to notify VA stakeholders of pending redirect.  

### Description
The H1 and url on this live page are changing: 
https://www.va.gov/resources/what-if-im-having-trouble-opening-a-pdf/

The H1 on this preview version of the page is now correct: 
http://preview-prod.vfs.va.gov/preview?nodeId=8684

### Requesting team info
*Requested by the Sitewide Content Team (Laura Walsh) and the DEPO Team (Beth Potts). 

### Implementation date
Date new URL(s) will be live: *We hope to have updated page published by the end of this week.*

Please indicate when the change/redirect(s) needs to be implemented"
- [ ] On the same date the new URLs launch
- [X] Can happen within 1-5 business days after new URLs launch
- [ ] Other - Please indicate timing : 

**The requesting team is responsible for notifying the group working on this issue if the target date changes. They are also responsible for ensuring the destination URLs are implemented correctly and live at the time the redirects are deployed.**

### Redirects needed

Current URL  |  Redirect Destination or New URL
---  |  ---
*https://www.va.gov/resources/what-if-im-having-trouble-opening-a-pdf/* | *https://www.va.gov/resources/how-to-download-and-open-a-pdf-form/*

### Definition of done
- [ ] Requesting team: Above information is provided and issue is tagged and assigned appropriately 
- [ ] Requesting team: All appropriate VA stakeholders are notified of pending redirect 
- [ ] VSP IA: Request is vetted and documented and implementation plan is clear - *@ VSP IA*
- [ ] VSP IA: Request is assigned to appropriate team for implementation - *@ VSP IA team*
- [ ] Implementation team: Implementation team completes work 
- [ ] Implementation team: Implementation pushed live and redirect is validated in production
- [ ] Requesting team: Ticket is closed

**Comment by lwalsh617** [comment]
Hi @mnorthuis -- just let me know if you need any more info on this. Thanks so much! 

FYSA - @bethpottsVADEPO

**Comment by mnorthuis** [comment]
Hi @lwalsh617 @bethpottsVADEPO,  because R&S is relatively new, and some of those pages do not receive a lot of traffic, I took a look at analytics to determine if a redirect is worth it.  In the case of this particular page, there are a few data points that lead me to feel that a redirect is not necessary for this page. 

- The traffic is extremely low - only 38 page views in the last month (I removed github visitors from all these stats)
- If you look at the "Entrances" column, this tells us how often this page was a landing page for users (i.e. the page users landed on when they first arrived at our site on), this can help us see how much traffic is coming directly to the page versus navigating to it through our site. This is important when it comes to redirects because external direct traffic is who we are trying to help/capture. 
  - Visitors arrived at va.gov on this page from google only 8 times (21% of page views) 
  - About 79% of the page views are from visitors who navigated to the page from somewhere else within va.gov (looks like almost all from other R&S pages)

In summary, the traffic to the page is very low, and of that traffic, most users are getting there from other pages on our site versus google searches.   So, implementing a redirect will ensure no users get a 404 when coming from google, but it won't transfer much search value, and we can remove the URL from google results through search console and removing it from our xml sitemap. 

Let me know your thoughts. 


Since there are likely no external direct links (excluding search engines), 

![image](https://user-images.githubusercontent.com/20994159/123839278-8de10c00-d8d2-11eb-8c95-761ae5a452e3.png)
![image](https://user-images.githubusercontent.com/20994159/123840547-0eecd300-d8d4-11eb-857c-a301a973199a.png)

**Comment by bethpottsVADEPO** [comment]
This is great @mnorthuis! Thank you for looking into it. And I agree--no need for a redirect. 

Thanks for inquiring, @lwalsh617!

**Comment by lwalsh617** [comment]
@mnorthuis - Thanks for writing this up in detail. Super helpful to understand your rationale!

**Comment by mnorthuis** [comment]
@bethpottsVADEPO @lwalsh617 Can you tell me what prompted a change to the H1 for this particular page?  

I think it may be helpful to put together some standards on when/why we change H1s and whether the change is valuable enough  (not saying this one isn't, just in general).   We want to keep H1s and URLs in R&S locked together, but the H1 should really be locked down once published unless there is a significant change, but that's where I'd love to hear input on what can possibly drive those changes and how we can best handle them.

**Comment by bethpottsVADEPO** [comment]
@mnorthuis It should be extremely rare to need to change the H1. In this case, we realized that the article and its H1 weren't framing the problem correctly to help Veterans with PDF forms. So we completely reworked it. Danielle and I are working on R&S standards and this will be included. @DanielleThierryUSDSVA for awareness

**Comment by bethpottsVADEPO** [comment]
@mnorthuis We'll publish the changes to this page this week. I don't think you're tracking R&S pages, but just wanted to let you know. Also, you mentioned removing the URL from google results through search console and removing it from our xml sitemap. Sounds like I should get in touch with John H. about that?

**Comment by mnorthuis** [comment]
@bethpottsVADEPO I'm not sure how the xml sitemap is updated with changes these days, but John H or Kelson would likely know.  I think as long as we ~remove~  change it in the sitemap we should be good (swap old URL for new URL).

**Comment by bethpottsVADEPO** [comment]
@RLHecht Would you open a request with public websites for this sitemap URL change?

**Comment by RLHecht** [comment]
New ticket #27034 created for public websites.

**Comment by RLHecht** [comment]
@lwalsh617 @bethpottsVADEPO I've just published this page with the new url.

**Comment by bethpottsVADEPO** [comment]
Thanks! I think we're good to close this ticket now.
